frontend/src/components/layout/AppNavBar/AppNavBar.test.tsx

```diff
@@ -46,55 +48,14 @@ describe('AppNavBar', () => {
     expect(mountToJson(tree.find(AppNavBar))).toMatchSnapshot();
   });
 
-  it('AppNavBar include View Projects Link', () => {
-    (useKeycloak as jest.Mock).mockReturnValue({
-      keycloak: {
-        subject: 'test',
-        userInfo: {
-          roles: ['admin-properties', 'admin-projects'],
-        },
-      },
-    });
-
-    const { getByText } = render(
-      <Provider store={store}>
-        <Router history={history}>
-          <AppNavBar />
-        </Router>
-      </Provider>,
-    );
-    const link = getByText('View Projects');
-    expect(link).toBeTruthy();
-  });
-
   describe('AppNavBar Links Based on Security', () => {
-    it('AppNavBar include Approval Requests Link', () => {
-      (useKeycloak as jest.Mock).mockReturnValue({
-        keycloak: {
-          subject: 'test',
-          userInfo: {
-            roles: ['dispose-approve'],
-          },
-        },
-      });
-      const { getByText } = render(
-        <Provider store={store}>
-          <Router history={history}>
-            <AppNavBar />
-          </Router>
-        </Provider>,
-      );
-      const link = getByText('Approval Requests');
-      expect(link).toBeTruthy();
-    });
-
     describe('AppNavBar Administation Dropdown', () => {
       it('AppNavBar include Administration Dropdown', () => {
         (useKeycloak as jest.Mock).mockReturnValue({
           keycloak: {
             subject: 'test',
             userInfo: {
-              groups: ['System Administrator'],
+              groups: Roles.SYSTEM_ADMINISTRATOR,
```

Comment:
changed this so future spelling issues wouldn't happen, like Jonny and I's 45-minute struggle

Reply:
:P

Reply:
this is changing groups from an array to a string though isn't it?

Reply:
oh yeah, I'll fix it